src/actions/__tests__/range-test.js

```diff
@@ -0,0 +1,24 @@
+import getRange from '../range'
+
+describe('actions/getRange', () => {
+  it('returns range between 2 and totalPage if totalPage' +
+     'is larger than preset range', () => {
+    expect(getRange(1, 1, 3)).toEqual([2])
+  })
+
+  it('returns range before and after current number according to set limit', () => {
+    expect(getRange(4, 10, 3)).toEqual([3, 4, 5])
+  })
+
+  it('sets initial to 2 if current number is 1', () => {
```

Comment:
Not quite the same. The first one test the scenario when the total page returned is less than our preset range, let's say, 3. Then, it should return the range between the current page and last page.